amoy.toml

```diff
@@ -352,6 +352,10 @@
 'v1-007500-007600-bodies.seg' = '95f14680ca199c9ee04157445c67d5974b7428c1'
 'v1-007500-007600-headers.seg' = 'a1027a170c6c8fd54591f90ab19241dd53443466'
 'v1-007500-007600-transactions.seg' = '94cb4123051a95ab3304dfaebc1e83acbef24e3c'
-'v1-007600-007700-bodies.seg' = '4c46cef63450865c1983f4c306325cb4f194995f'
```

Comment:
hmm something changed. i will try re-gen this file.

Reply:
OK,so I'm closing this PR.